iulianIachimov/src/test/java/generics/GenericsLibraryTest.java

```diff
@@ -0,0 +1,62 @@
+package generics;
+
+import genericstask.model.Books;
+import genericstask.model.GenericsLibrary;
+import genericstask.model.Newspapers;
+import genericstask.model.Video;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+
+public class GenericsLibraryTest {
```

Comment:
please remove public, same for tests

Reply:
I am afraid it cannot be done as when removing public the tests become impossible to execute. Please advise me if I am getting it wrong.

Reply:
please ignore, as you use junit 4